src/add/operator/first.ts

```diff
@@ -2,6 +2,6 @@
 import {Observable} from '../../Observable';
 import {first} from '../../operator/first';
 
-Observable.prototype.first = first;
+Observable.prototype.first = <any>first;
```

Comment:
It lets you supply null / undefined and skip the `R` argument. [see example](http://www.typescriptlang.org/Playground#src=interface%20Observable%3CT%3E%20%7B%7D%0Aexport%20interface%20FirstSignature%3CT%3E%20%7B%0A%20%20(predicate%3F%3A%20(value%3A%20T%2C%20index%3A%20number%2C%20source%3A%20Observable%3CT%3E)%20%3D%3E%20boolean)%3A%20Observable%3CT%3E%3B%0A%20%20(predicate%3A%20(value%3A%20T%2C%20index%3A%20number%2C%20source%3A%20Observable%3CT%3E)%20%3D%3E%20boolean%2C%20resultSelector%3A%20void%2C%20defaultValue%3F%3A%20T)%3A%20Observable%3CT%3E%3B%0A%20%20%3CR%3E(predicate%3F%3A%20(value%3A%20T%2C%20index%3A%20number%2C%20source%3A%20Observable%3CT%3E)%20%3D%3E%20boolean%2C%20resultSelector%3F%3A%20(value%3A%20T%2C%20index%3A%20number)%20%3D%3E%20R%2C%0A%20%20%20%20%20%20defaultValue%3F%3A%20R)%3A%20Observable%3CT%20%7C%20R%3E%3B%0A%7D%0A%0Alet%20f%3A%20FirstSignature%3Cnumber%3E%3B%0A%0Alet%20s%20%3D%20f(x%20%3D%3E%20x%20%3D%3D%3D%201%2C%20null%2C%200)%3B%0Alet%20b%20%3D%20f(x%20%3D%3E%20x%20%3D%3D%3D%201%2C%20x%20%3D%3E%20x.toString()%2C%20'7')%3B%0A%0A%0A%0A).
 Now it does look like that last overload shouldn't be `<T | R>` but just `<R>`, I'll try and fix that tomorrow morning.